### PR TITLE
Add Autocomplete icon

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Screenshot from 2020-09-07 18-25-56

The icon doesn't look optimal on a dark theme. @asaadmahmood I'm wondering what you think about this.

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-plugin-github/issues/358

### CODE REVIEW

File: server/plugin/command.go [hunk 1]
```diff
@@ -60,16 +64,20 @@ func validateFeatures(features []string) (bool, []string) {
 	return valid, invalidFeatures
 }
 
-func getCommand(config *Configuration) *model.Command {
-	return &model.Command{
-		Trigger:          "github",
-		DisplayName:      "GitHub",
-		Description:      "Integration with GitHub.",
```
Comment (hanzei) [inline server/plugin/command.go:67]: `DisplayName` and `Description` are not needed for plugins as they are not shown.

**Comment by hanzei** [comment]
Heads up: The code might be replaced by the one in https://github.com/mattermost/mattermost-plugin-api/pull/71 if the PR gets accepted.

**Comment by asaadmahmood** [comment]
@hanzei We should use the same github image as we use for the Github bot.
The Github bot image that we use has a white background behind it.

Screenshot 2020-09-08 at 12 34 51 PM

**Comment by hanzei** [comment]
@asaadmahmood In the Marketplace modal we put a white background behind every icon. I was wondering we we want to do the same here and don't task the developer with providing the background.

**Comment by asaadmahmood** [comment]
@hanzei We can do that, but we would still have to use a different image that has a padding built in. So that the image is not too close to the edges of the white background.
I would lean towards not having a white background, just in case people intentionally want to keep the bg transparent.

**Comment by hanzei** [comment]
Do we have to have a different image? This is how it looks in the Marketplace with the same svg used as in this PR.
Screenshot from 2020-09-08 16-39-10

**Comment by asaadmahmood** [comment]
@hanzei Here's the image in our Github bot avatar, we can use this.
image

**Comment by hanzei** [comment]
As per request from @asaadmahmood I've updated the icon to use one with a white background:
Screenshot from 2020-09-08 17-43-51
Screenshot from 2020-09-08 17-43-37

**Review by iomodo — APPROVED** [review]

**Review by larkox — APPROVED** [review]

**Comment by hanzei** [comment]
@asaadmahmood Does this look good to you?

**Review by asaadmahmood — APPROVED** [review]
Yup, looks good.

**Comment by DHaussermann** [comment]
- Icon always displays as expected
- Checked icon looks good on all themes
- Tested on desktop and various browsers

@hanzei This change is working but, something is odd about the unit tests. Can you please take a look?
```
Test Suites: 1 passed, 1 total
Tests:       2 passed, 2 total
Snapshots:   0 total
Time:        4.118 s
Ran all test suites.
cd ./build/sync && /usr/local/go/bin/go test -v -race ./...
go: warning: "./..." matched no packages
no packages to test
make: *** [test] Error 1
```

**Comment by hanzei** [comment]
@DHaussermann https://github.com/mattermost/mattermost-plugin-github/pull/363 will fix the failure in `make test`. That issue is unrelated to this PR.

**Review by DHaussermann — APPROVED** [review]
Tested and Passed
Only one issue was found in testing. It is unrelated and will be addressed separately. 

LGTM!

**Comment by hanzei** [comment]
Waiting for https://github.com/mattermost/mattermost-plugin-github/pull/363 to be merged first.

**Pull request merged**